vmware/ngc/NGC-Register/src/main/resources/templates/homePage.html

```diff
@@ -0,0 +1,151 @@
+        <img src="/icon/openSDS.png" style="width:205px; height:75px; text-align: center;">
+        <h1 > vSphere Plugin Register Page</h1>
+    </div>
+    <br>
+    <div>
+        <li>If you want to
+            <span th:if="${VCenterInfo.registerFlag == '0'}" style="color: #428bca;">register</span>
+            <span th:if="${VCenterInfo.registerFlag == '1'}" style="color: #d9534f;">unregister</span>
+            the Next Generation Client(NGC) plugin, please provide a valid vCenter Server IP, user name and password.
+        </li>
+    </div>
+    <div calss="divider"></div>
+    <hr style="margin-top: 15px; margin-bottom: 30px;">
+    <center>
+        <form id="ngcForm"  class="form-horizontal" onkeydown="if(event.keyCode==13){return false;}"
+              accept-charset="utf-8" method="post" th:action="@{/action}" th:object="${VCenterInfo}">
+            <div class="form-group" style="width:600px">
+                <label class="col-sm-4 control-label" for="vCenterIp" >vCenter server ip</label>
+                <div class="col-sm-8">
+                    <input type="text" class="form-control" id ="vCenterIp" th:field="*{vCenterIp}"
+                           placeholder="input vcenter ip" name ="vCenterIp" >
+                </div>
+            </div>
+            <br>
+            <div class="form-group" style="width:600px">
+                <label class="col-sm-4 control-label" for="vCenterUser">User name</label>
+                <div class="col-sm-8">
+                    <input type="text" class="form-control" id ="vCenterUser"
+                           placeholder="input vcenter username" name ="vCenterUser"  data-bv-m>
+                </div>
+              <!--  <label style="color:red" th:if="${#fields.hasErrors('vCenterUser')}" th:errors="*{vCenterUser}">Name Error</label>-->
+            </div>
+            <br>
+            <div class="form-group" style="width:600px">
+                <label class="col-sm-4 control-label" for="vCenterPassword">User Password</label>
+                <div class="col-sm-8">
+                    <input type="password" class="form-control" id ="vCenterPassword"
+                           placeholder="input vcenter password" name ="vCenterPassword">
+                </div>
+               <!-- <label style="color:red" th:if="${#fields.hasErrors('vCenterPassword')}" th:errors="*{vCenterPassword}">Name Error</label>-->
+            </div>
+            <hr style="margin-top: 30px; margin-bottom: 15px;">
+            <div th:if="${VCenterInfo.registerFlag == '1'}" >
+                <li style="text-align: left; margin-bottom: 15px;">The NGC Plugin has already registered in
+                    <span th:text="*{vCenterIp}" style="color:#ac2925;"> </span>.
+                </li>
+            </div>
+            <div class="form-group" style="">
+                <button type="submit" name="action" class="btn btn-primary" th:if="${VCenterInfo.registerFlag == '0'}"
```

Comment:
Do we use vCenter or VCenter ? Can we make it uniform? If any specific reason for VCenter, please mention.

Reply:
"vCenter" will appear on the webpage. Fixed.